### PR TITLE
Add a minimal EULA check (CAPS-117)

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

File: harbor-helm/templates/NOTES.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
+{{- required "\n\nEULA not accepted.\nPlease review https://documentation.suse.com/beta/eula and to accept, add --set suse.AcceptBetaEULA=y on your helm invocation" .Values.suse.AcceptBetaEULA }}
```
Comment (flaviodsr) [inline harbor-helm/templates/NOTES.txt:1]: I think the error message without the newlines looks much better even comparing to using `fail`

File: harbor-helm/values.yaml [hunk 2]
```diff
@@ -1,3 +1,10 @@
+suse:
+  # distributed under the SUSE(R) End User License Agreement for Beta Software
+  # please review + accept:
+  # https://documentation.suse.com/beta/eula
+  AcceptBetaEULA:
+    ""
```
Comment (flaviodsr) [inline harbor-helm/values.yaml:6]: I don't think this line is necessary.

**Review by flaviodsr — CHANGES_REQUESTED** [review]

File: harbor-helm/values.yaml (hunk 2 above)
Comment (dirkmueller) [reply harbor-helm/values.yaml]: removed

File: harbor-helm/templates/NOTES.txt (hunk 1 above)
Comment (dirkmueller) [reply harbor-helm/templates/NOTES.txt]: ok, removed newlines again

**Review by flaviodsr — APPROVED** [review]

**Review by flaviodsr — APPROVED** [review]

**Review by jsuchome — APPROVED** [review]

**Pull request merged**